CONTRIBUTING.md

```diff
@@ -321,3 +321,42 @@ stripped with no prior warning or consent from the member in question.
 Earning this privilege should be considered to be no small feat and is by no
 means guaranteed by any quantifiable metric. It is a symbol of great trust of
 the community of this project.
+
+## Point Release Procedure
```

Comment:
```suggestion
## ADR Process
```

@okwme this is my recommendation. The section's title sounds wrong to me.

Reply:
@okwme's intention I believe is to document how we approve a release, but I agree that the content appears very focused on ADRs.

Reply:
Yep, that was also one of my points in the [original issue](https://github.com/cosmos/cosmos-sdk/issues/7490#issuecomment-708972871):

```
* RFC process should be distinct from Release Management.
```

RFC is about strategic technical direction. Release Management and Stable Release Management should have clear and fairly unflexible processes by which Release Manages must abide by.